restate-test-utils/src/test_utils.rs

```diff
@@ -0,0 +1,174 @@
+
+use restate_admin_rest_model::deployments::RegisterDeploymentRequest;
+use testcontainers::{core::{IntoContainerPort, WaitFor}, runners::AsyncRunner, ContainerAsync, ContainerRequest, GenericImage, ImageExt};
+use serde::{Serialize, Deserialize};
+use restate_sdk::{discovery::Service, errors::HandlerError};
+use std::time::Duration;
+
+#[derive(Serialize, Deserialize, Debug)]
+struct VersionResponse {
+    version:String,
+    min_admin_api_version:u32,
+    max_admin_api_version:u32
+}
+
+
+pub struct TestContainer {
```

Comment:
It would be nice if the test infra bootstraps the service endpoint server too, and then on `Drop` stops the server.